### ISSUE TITLE
Able to manage split XCI files.

### ISSUE THREAD

**Issue by KaLoF87** [gibaBR/Switch-Backup-Manager#3]
Some people are using fat32 on their microSD cards and need to split games that are over 4gb.
Games that are splitted doesnt get added in the list. 

It would be great if that can be fixed/added. :)

**Comment by gibaBR** [comment]
Please, test with this binary:
https://mega.nz/#!LC5HwSYa!HAACFTFDnedP32Lucc-cQ2DKwQmxpXLcwkO0DPOYAzo
For now it works only when you do add ->Files...
Select the first split (xc0)

**Comment by KaLoF87** [comment]
Great! I can now add the splitted games to the backup manager.
I noticed a small bug: when I add one file, the scraping progress bar doesnt say the name of the game.
When I add the second game the progress bar will show the name of the first game I added. 
So it will only show the name of the previous game that I added.

In the future when you have time!
It would be great to maybe add a tab which say if the xci is splitted and how many files there is.

Really nice work with the manager! I think this will be a awesome tool to use when managing the games on the Switch!
